plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -234,7 +234,10 @@ def process_main(self):
                 if content_type == man_list:
                     self._process_manifest_list(manifest, digest, available_blobs, tag)
                 else:
-                    self._process_manifest(manifest, digest, available_blobs, tag)
+                    foreign_layers = self._process_manifest(manifest, digest, available_blobs, tag)
```

Comment:
Just looking at this line, I would reasonably suspect that ``foreign_layers`` is a list of layers. Since it's actually a bool, I would use the same name you did in the method and call this ``has_foreign_layer``.